Orange/tests/test_metadata_save.py

```diff
@@ -0,0 +1,36 @@
+# Test methods with long descriptive names can omit docstrings
```

Comment:
Is there a special reason this tests were not placed in `Orange/tests/test_tab_reader.py`? They only test TabReaders functionality and all other TabReader tests are in that file.